python/runtime/model/model.py

```diff
@@ -0,0 +1,95 @@
+# Copyright 2020 The SQLFlow Authors. All rights reserved.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""This module saves or loads the SQLFlow model.
+"""
+import os
+from enum import Enum
+
+# TODO(yancey1989): move diag to the upper direction
```

Comment:
direction -> directory?

Reply:
The name `diag` is confusing, maybe use `diagnostic.py`?

Reply:
> The name diag is confusing, maybe use diagnostic.py?
I will do that in the next PR.